# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/moz.build
# Commit: 802512ee57ae
# Full Hash: 802512ee57aec9d975ab078c48b9f37956012bc6
# Author: Neil Deakin <neil@mozilla.com>
# Date: 2015-04-17 19:45:37
# Regressor Bug: 1071562
# File Overlap Count: 1
# Description:
#   Bug 1071562, e10s, support non-text types in clipboard (html, images, etc), r=smaug
# ==============================================================================

diff -r ffd1b16f058b -r 802512ee57ae dom/base/moz.build
--- a/dom/base/moz.build	Thu Apr 16 01:11:05 2015 +0300
+++ b/dom/base/moz.build	Thu Apr 16 15:38:12 2015 -0400
@@ -417,6 +417,7 @@
     '/dom/xml',
     '/dom/xslt/xpath',
     '/dom/xul',
+    '/gfx/2d',
     '/image/src',
     '/js/xpconnect/src',
     '/js/xpconnect/wrappers',